pkg/api/types.go

```diff
@@ -1595,6 +1595,45 @@ type Handler struct {
 	TCPSocket *TCPSocketAction
 }
 
+// DeleteExecAction describes a "run in container" action that will be invoked
+// inside of a container prior to sending the TERM signal to the container's
```

Comment:
It means that if you have a pod with multiple containers, the PID namespace is shared among all the containers. Before this change, each container's PID 1 was its entrypoint, and if you killed PID 1 in a single container, that container would die, but the other containers in the pod would be unaffected. With a shared PID namespace, PID 1 is owned by the pod sandbox (is that the right terminology @derekwaynecarr?), and each container's entrypoint has some unique PID other than 1. If you kill PID 1 in the shared PID namespace, you will bring down all the containers for that pod.